client/components/domains/search-filters/tld-filter-bar.jsx

```diff
@@ -112,6 +131,31 @@ export class TldFilterBar extends Component {
 		);
 	}
 
+	renderSuggestedCheckboxes() {
+		const {
+			lastFilters: { tlds: selectedTlds },
+		} = this.props;
+		const checkboxes = this.props.availableTlds
+			.slice( 0, this.props.numberOfTldsShown )
+			.map( ( tld, index ) => (
+				<div className="search-filters__tld-checkbox">
```

Comment:
@niranjan-uma-shankar Could you please add a unique `key` to these components? They are throwing a warning in the console:
```
backend.js:6 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `TldFilterBar`.
```

🙏